.github/scripts/instr-count-benchmark/run-benchmark.sh

```diff
@@ -13,3 +13,6 @@ PYTORCH_GIT_VERSION=$(python -c "import torch; print(torch.version.git_version)"
 echo "Running instruction benchmark for pytorch-${PYTORCH_VERSION}, commit SHA: ${PYTORCH_GIT_VERSION}"
 
 # Run the instruction count benchmark
+pushd ${REPO_CHECKOUT}
```

Comment:
But then shouldn't we first enter the directory?
```suggestion
pushd ${BENCHMARK_ROOT}
```

Reply:
Ah! You are correct.